e2e/utils/wallet.ts

```diff
@@ -0,0 +1,15 @@
+import { entropyToMnemonic } from '@ethersproject/hdnode';
+import { Wallet, randomBytes } from 'ethers';
+
+export function generateRandomRecoveryPhrase() {
+  const entropy = randomBytes(16);
+  return entropyToMnemonic(entropy);
+}
+
+export function generateRandomWallet() {
```

Comment:
I think this should conform to wallets that are being created in `src/shared/wallet/create.ts`:
```suggestion
export function generateRandomWallet(): BareWallet {
```

Reply:
updated ✅